pkg/after_dhtnode_install.sh

```diff
@@ -5,15 +5,14 @@ if [ "$1" = "configure" ]; then
     adduser --system --no-create-home dhtnode
 
     # Check that deployment directory exists
-    test -d /srv/dhtnode || exit 1
+    test -d /srv/dhtnode || (echo "/srv/dhtnode/dhtnode-* directories missing"; exit 1)
```

Comment:
I'm not sure if `exit 1` is a correct behaviour for binary package, as it will still make configuring package by `dpkg` to fail.

Reply:
That's the question for other PR/issue. This PR just tidies up the the loose ends.

Reply:
(noncontroversal ends)

Reply:
What would you suggest instead? (This PR is just about fixing up what we currently have, not about the wider changes that were discussed.)